java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1744,6 +1744,23 @@ public ColumnVector endsWith(Scalar pattern) {
     }
   }
 
+  /**
+   * Checks if each string in a column contains a specified comparison string, resulting in a
+   * parallel column of the boolean results.
+   * @param pattern scalar containing the string being searched for.
+   * @return A new java column vector containing the boolean results.
+   */
+
+  public ColumnVector stringContains(Scalar pattern) {
```

Comment:
nit: I might be missing something but looking at the cpp header, its just a regular string, calling it a pattern might throw people off?

Reply:
This keeps it in line with startswith and endswith implementations. Let me know if that works or I'll go ahead and change all three although that would expand the footprint of the patch.

Reply:
I do think it should be changed but I will leave that up to you.

Reply:
LGTM otherwise

Reply:
Thanks Reza for the feedback. I would keep it this way and maybe do a refactor PR separately to keep commit annotations clean.

Reply:
Hmm. @razajafri might have a point here. I'd think `pattern` has regex-like connotations.

Reply:
I agree, I did not want to change `startsWith` and `endsWith` in a PR which adds `contains`. Let me just change it for the method I am working on and do the others as a follow up. Thanks @razajafri / @mythrocks